io.openems.edge.common/src/io/openems/edge/common/powercharacteristic/package-info.java

```diff
@@ -0,0 +1,3 @@
+@org.osgi.annotation.versioning.Version("1.0.0")
+@org.osgi.annotation.bundle.Export
+package io.openems.edge.common.powercharacteristic;
```

Comment:
What could be a generic name for this? How about `linecharacteristic` or `polyline` (see e.g. https://www.w3schools.com/graphics/svg_polyline.asp)